projects/valtimo/document/src/lib/document.service.ts

```diff
@@ -270,6 +270,19 @@ export class DocumentService {
     );
   }
 
+  unassignHandlerFromDocument(documentId: string): Observable<any> {
```

Comment:
Type any; this cannot be typed? It does not return anything? Then it should be Observable<void>